oneview/resource_enclosure_group.go

```diff
@@ -46,10 +47,11 @@ func resourceEnclosureGroup() *schema.Resource {
 			"description": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
```

Comment:
Is this getting auto-populated?
Setting Description to computed+optional means you cant set it to null in the update operation. 
I saw a use case scenario in REST Guide, so raised this comment.